.travis.yml

```diff
@@ -8,6 +8,9 @@
 
 addons:
   postgresql: 9.4
+  apt:
+    packages:
+    - rabbitmq-server
```

Comment:
@topless @roksys this is the fix for the build.
See https://docs.travis-ci.com/user/database-setup/#rabbitmq

It was hanging doing `flask index queue init` [here](https://github.com/inveniosoftware/invenio-oaiserver/blob/master/examples/app-setup.sh#L20) because it was waiting for `rabbitmq`.